classes/author.rb

```diff
@@ -0,0 +1,15 @@
+class Author
+  attr_accessor :name, :items
+  attr_reader :id
+
+  def initialize(name)
+    @id = rand(1...1000)
+    @name = name
```

Comment:
First name and Last names should be separately stored in two individual variables